examples/browser-webrtc/src/lib.rs

```diff
@@ -19,14 +16,13 @@ pub async fn run(libp2p_endpoint: String) -> Result<(), JsError> {
     let body = Body::from_current_window()?;
     body.append_p("Let's ping the WebRTC Server!")?;
 
-    let swarm = libp2p::SwarmBuilder::with_new_identity()
+    let mut swarm = libp2p::SwarmBuilder::with_new_identity()
         .with_wasm_bindgen()
         .with_other_transport(|key| {
-            webrtc_websys::Transport::new(webrtc_websys::Config::new(&key))
+            libp2p_webrtc_websys::Transport::new(libp2p_webrtc_websys::Config::new(&key))
         })?
         .with_behaviour(|_| Behaviour {
             ping: ping::Behaviour::new(ping::Config::new()),
-            keep_alive: keep_alive::Behaviour,
         })?
         .build();
```

Comment:
You'll have to use `with_swarm_config` here and set the idle connection timeout. See some of the other examples for how to exactly.